index.html

```diff
@@ -64,36 +63,37 @@
           // appropriately mapping on the core-aam spec.
           coreMappingURLs: {
               "ED":   "https://w3c.github.io/core-aam/",
-              "WD":   "https://www.w3.org/TR/core-aam-1.1/",
+	      "CR" :  "https://w3c.github.io/core-aam/",
+              "WD":   "https://w3c.github.io/core-aam/",
               "FPWD": "https://www.w3.org/TR/core-aam-1.1/",
               "REC":  "https://www.w3.org/TR/wai-aria-implementation/"
           },
 
 	    preProcess:[linkCrossReferences],
 		postProcess:[],
 		definitionMap:[],
-		xref: ["core-aam", "accname", "wai-aria"]
+		xref: ["dom", "core-aam", "accname", "wai-aria", "infra"]
```

Comment:
This is the reason that the references failed, they didn't know to look in "dom" and "infra"